app/frontend/src/theme.ts

```diff
@@ -87,7 +86,7 @@ const themeOptions: ThemeOptions = {
   shape: {
     borderRadius,
     navPaddingDesktop: spacing(10),
-    navPaddingMobile: spacing(navBarHeight + navBarPadding),
+    navPaddingMobile: spacing(navBarHeight),
```

Comment:
Can you also delete the variable from constants.ts or wherever it is?

Reply:
It was in this file on line 18
unless you mean something else

Reply:
Sorry I missed it